bundles/CoreBundle/Resources/migrations/Version20200210101048.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Pimcore\Migrations\Migration\AbstractPimcoreMigration;
+use Pimcore\Model\DataObject\ClassDefinition;
+
+class Version20200210101048 extends AbstractPimcoreMigration
+{
+    public function doesSqlMigrations(): bool
+    {
+        return false;
+    }
+
+
+    /**
+     * @param Schema $schema
+     * @throws \Exception
+     */
+    public function up(Schema $schema)
+    {
+        $list = new ClassDefinition\Listing();
+        $list = $list->load();
+
+        foreach ($list as $class) {
+            if ($class->getCacheRawRelationData()) {
```

Comment:
This check is obsolete, the method `getCacheRawRelationData()` should go anyway, even if we keep it it always returns `true`